langchain/chains/llm.py

```diff
@@ -43,6 +48,8 @@ def lc_serializable(self) -> bool:
     """Prompt object to use."""
     llm: BaseLanguageModel
     output_key: str = "text"  #: :meta private:
+    output_parser: BaseLLMOutputParser = Field(default_factory=DefaultOutputParser)
+    return_final_only: bool = True
```

Comment:
It's not obvious what this parameter does until one reads the code where the parameter is used (which doesn't make for an obvious API).  Either a better variable name or a doc-string (or both) would help here. 

```
return_final_only: bool = True
"""If true, returns only the parsed generations for each input. If false, will return the parsed information together with the raw completions from the llm for each input."""
```

It might be worth to expand the doc-string for LLMChain.